man/dnstable_lookup.1.txt

```diff
@@ -45,9 +47,30 @@ can be used to request a specific type's version. ^ETYPE^ may be one of ^rrset^,
 
 == OPTIONS ==
 
-^-R^, ^--raw^::
-    When using a JSON format for output, add a hex representation of the
-    raw rdata and the rrtype.
+^-a^ 'datetime'::
+    Select results with recorded first seen after this 'datetime'.
+    See the DATE TIME SPECIFICATION section below for details.
+
+^-A^ 'datetime'::
+    Select results with recorded last seen after this 'datetime'.
+    See the DATE TIME SPECIFICATION section below for details.
+
+^-b^ 'datetime'::
+    Select results with recorded last seen before this 'datetime'.
+    See the DATE TIME SPECIFICATION section below for details.
+
+^-B^ 'datetime'::
+    Select results with recorded first seen before this 'datetime'.
+    See the DATE TIME SPECIFICATION section below for details.
+
+^-c^::
+    This enables complete (strict) matching semantics so no entries with
+    a first or last observed time outside the provided ^-a^, ^-A^, ^-b^,
+    and/or ^-B^ time range will be matched.
+    By default, time-based matching will have results if any of the
```

Comment:
Since `-c` only changes filters with `-A` and `-B`, I'd suggest removing references to `-a` and `-b` here, and starting the sentence "By default, time-based matching with `-A` and `-B`..."

Also, consider "time-based filtering ... will select results" as alternate wording.